rkt/image/fetchers/common.go

```diff
@@ -0,0 +1,49 @@
+package fetchers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type InsecureOpts struct {
+	AllowHTTP      bool `json:"allow_http"`
+	SkipTLSCheck   bool `json:"skip_tls_check"`
+	SkipImageCheck bool `json:"skip_image_check"`
+}
+
+type FetcherConfig struct {
```

Comment:
General concern: We have to provide a strong guarantee about the long-term stability of this fetcher config. Adding/removing/changing fields will be a major concern.

Maybe add a `Version` config field to be more flexible?